feat(linux): Add the documentation for UMS command

hrushikesh221 opened the pull request:
Add the documentation for UMS(USB Mass Storage) command.

StaticRocket commented on source/linux/Foundational_Components/U-Boot/UG-Memory-K3.rst:
```diff
@@ -242,6 +242,73 @@ to copy the boot images:
 
+.. ifconfig:: CONFIG_part_variant in ('J721E','J784S4', 'J742S2', 'J721S2', 'J7200', 'J722S', 'AM62X', 'AM62AX', 'AM62LX', 'AM62PX')
+
+   Accessing an SD card from a host PC using UMS command
+   =====================================================
```
https://github.com/TexasInstruments/processor-sdk-doc/blob/master/CONTRIBUTING.md#behavior

This should be in a separate file if it's platform specific section.

hrushikesh221 replied:
@StaticRocket  Understood. However UMS command is enabled for all the platforms except AM68 and AM69 as they don't support USB peripheral mode of operation. Hence UMS documentation is applicable to all the platforms except this two platforms. So should I still create new file for UMS?

hrushikesh221 commented:
@StaticRocket I addressed some of the vale comments. Remaining warnings are asking to define the abbreviations like MMC, SATA, UMS. And to check the words like devnum, partnum as it is not able to find in the dictionary.

hrushikesh221 commented:
@praneethbajjuri I addressed the comments from @StaticRocket, after that I lost track of it. Seems like there are merge conflicts, I will address those and update the PR.

hrushikesh221 commented:
@StaticRocket Thanks for your feedback. I will update the PR accordingly.
